submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/security/SecurityApiTest.kt

```diff
@@ -74,6 +79,28 @@ class SecurityApiTest(
         assertThat(user.email).isEqualTo("case-insensitive-inactive@mail.org")
     }
 
+    @Test
+    fun `22-6 check ftp home type user`() =
```

Comment:
These tests should be added to the [inventory](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/itest/resources/itestsInventory/itestsInventory.md)